src/object.c

```diff
@@ -957,12 +957,12 @@ char *strEncoding(int encoding) {
  * Actually the number of nodes and keys may be different depending
  * on the insertion speed and thus the ability of the radix tree
  * to compress prefixes. */
-size_t streamRadixTreeMemoryUsage(rax *rax) {
-    size_t size;
-    size = rax->numele * sizeof(streamID);
-    size += rax->numnodes * sizeof(raxNode);
+size_t streamRadixTreeMemoryUsage(rax *r) {
+    size_t size = sizeof(r);
+    size = r->numele * sizeof(streamID);
+    size += r->numnodes * sizeof(raxNode);
     /* Add a fixed overhead due to the aux data pointer, children, ... */
-    size += rax->numnodes * sizeof(long)*30;
+    size += r->numnodes * sizeof(long)*30;
```

Comment:
reminds me of my first-ever PR :)
https://github.com/redis/redis/pull/3579

Reply:
ok, but we can do that with less changes
```c
size_t size = sizeof(*rax);
```
or 
```c
size_t size = sizeof(struct rax);
```

Reply:
:angry:

Reply:
jk, i'll do it